bridge/bridge.go

```diff
@@ -27,6 +27,13 @@ type Opts struct {
 	Lambda bool   `json:"Lambda"`
 }
 
+// ChainlinkRequest is the format that the adapter receives
+// data from the Chainlink node
+type ChainlinkRequest struct {
```

Comment:
Rather than creating a new struct, add `ID` into the `Result` struct, and then just set the value during a request.